test/chpldoc/linkage-spec/exportTest.doc.catfiles

```diff
@@ -1,2 +1,2 @@
-docs/exportTest.doc.txt
-docs/exportTest.doc/Other.txt
+docs/_sources/modules/exportTest.doc.txt
+docs/_sources/modules/exportTest.doc/Other.txt
```

Comment:
We still get the docs directory.  Because I'm not throwing anything explicitly, this is also generating html, but it is saving the source files it used for that html into the subdirectory `_sources/modules`.  I believe it is our choice for that extension to be .txt and it is probably not difficult to change, but I refrained in this PR in an effort to keep it a "pure test" change.